bin/jwtwebhook.py

```diff
@@ -319,13 +305,18 @@ def run(self, stanza, cleaned_params, input_config):
             else:
                 path_re = None
 
-            def output_results(results):
-                """
-                This function will get the web-server to output the results to Splunk.
-                """
-                for result in results:
-                    self.output_event(result, stanza, index=index, source=source, sourcetype=sourcetype, host=host,
-                                      unbroken=True, close=True)
+            # Construct Splunk Event and Write to Splunk
+            def output_results(payload):
+                event = Event(
+                data=json.dumps(payload),
+                time="%.3f" % time.time(),
+                index=index,
+                source=source,
+                sourcetype=sourcetype,
+                done=True,
+                unbroken=True
```

Comment:
stylistically... would we want to indent these so that they are visibly part of the Event invocation?

Reply:
I’ll make some cosmetic changes lol